tests/sqpparser_clickhouse.rs

```diff
@@ -40,16 +41,32 @@ fn parse_map_access_expr() {
                     )))),
                     FunctionArg::Unnamed(FunctionArgExpr::Expr(Expr::Value(
                         Value::SingleQuotedString("endpoint".to_string())
-                    )))
+                    ))),
                 ],
                 over: None,
-                distinct: false
-            })]
+                distinct: false,
+            })],
         },
         expr_from_projection(only(&select.projection)),
     );
 }
 
+#[test]
+fn parse_array_expr() {
```

Comment:
Can you please also add a test for the `named` variant.

I think you added support for `SELECT ARRAY['1', '2'])` as well but it isn't covered in the tests

Reply:
named array has been tested in sqlparser_postgres.rs, so I didn't test named array separately here。

Reply:
Found it https://github.com/sqlparser-rs/sqlparser-rs/blob/1da49c1/tests%2Fsqlparser_postgres.rs#L826-L843 👍 

Thanks